runtime/runtime/src/congestion_control.rs

```diff
@@ -115,22 +106,21 @@ impl ReceiptSink {
             trie,
             outgoing_buffers.shards(),
             ReceiptGroupsConfig::default_config(),
-            apply_state.current_protocol_version,
         )?;
 
         let mut stats = ReceiptSinkStats::default();
         stats.set_outgoing_limits(
             outgoing_limit.iter().map(|(shard_id, limit)| (*shard_id, (limit.size, limit.gas))),
         );
 
+        let bandwidth_scheduler_output = Some(bandwidth_scheduler_output);
```

Comment:
likely should fix this too

Reply:
Seems somewhat easier to leave this as is vs. modifying ReceiptSinkV2 to take non-optional.

Reply:
I think here we can change this to `BandwidthSchedulerOutput` without the option. In runtime/runtime/src/lib.rs where the value is being set to None, we can just change it to default.

Reply:
tbh the issue here is the construction of a default `BandwidthSchedulerOutput`, I am not sure how to initialize the [`BandwidthSchedulerParams`](https://github.com/near/nearcore/blob/218efc0f994336cd089fbd6611ec93ec65bf8276/runtime/runtime/src/bandwidth_scheduler/mod.rs#L27)

I made an attempt here: https://github.com/near/nearcore/pull/13402/commits/98d054f9d713ad43ff2bbe1d09172103244e1b77 but I am not sure if that's the correct way to construct the BandwidthSchedulerParams (even though it's unused seems odd to construct it incorrectly).

Another alternative is to pass in the params as an argument when its used and only initialize the hash to a default. I also tried this but it's more of a refactor.

Reply:
Current system looks good as discussed offline!